src/types/metatype_keyT.ts

```diff
@@ -21,13 +21,13 @@ const metatypeKeyOptional = t.partial({
     id: t.string,
     archived: t.boolean,
     metatype_id: t.string,
-    cardinality: t.number,
+    cardinality: t.number, // not entirely sure why we have this yet
```

Comment:
Does this need to be resolved/determined before merge?

Reply:
Same for unique property below

Reply:
I've thought about it some more. Cardinality is currently just set to 1 for the ontology extractor, but may be used in the future for keys. Same for unique, looks like it's currently set to true for all keys but that may need to be adjusted to more accurately represent keys in the future. So I think they should both be there.